microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Data and Privacy in WebView2

```

Comment:
```suggestion
title: Data and privacy in WebView2
```
per corp style guides, use sentence case for titles/headings.  Unless you want "Privacy" as a special exception like What's New or Release Notes.